src/components/Dropdown/ButtonDropdown.tsx

```diff
@@ -22,20 +22,25 @@ export interface ButtonDropdownProps extends Partial<Omit<DropdownMenuProps, "tr
 }
 
 export const ButtonDropdown = ({
-  children,
-  isDisabled,
-  dataQa,
-  buttonAppearance,
-  buttonClassName,
-  ...props
-}: ButtonDropdownProps) => {
+                                 children,
```

Comment:
fix these spaces?

Reply:
It's like this after formatting, do you have any suggestions, I use webstorm

Reply:
i think it should only need two spaces just like the others